net.solarnetwork.node.setup.web/web/WEB-INF/jsp/a/home.jsp

```diff
@@ -58,6 +70,8 @@
 </section>
 
 <section id="datum-activity" class="hide">
+	<h2>Datum Graphs</h2>
```

Comment:
This graph area should be its own `<section>` and it probably makes sense to position above the `datum-activity-seenprops` section. All string literals must be localized by moving into the `/web/WEB-INF/messages.propertes` bundle and then rendered via `<fmt:message/>` JSP tags. There are existing messages starting with `datum.activity` at the end of that file, so it makes sense to add a new section using `datum.charts`, for example

```
datum.charts.title = Datum Charts
```

I think  the term _charts_ makes more sense than _graphs_ for the target audience of this feature, so I've used that term here.